src/modules/connthrottle.c

```diff
@@ -89,6 +91,7 @@ MOD_TEST()
 	safe_strdup(cfg.reason, "Throttled: Too many users trying to connect, please wait a while and try again");
 	cfg.minimum_reputation_score = 24;
 	cfg.sasl_bypass = 1;
+	cfg.webirc_bypass = 1; /* I am not sure... probably disabled with 0 by default? */
```

Comment:
AFAIK this doesn't actually set the default, it just tests the config if the default was 1? (not sure)

Reply:
I think that actually that MOD_TEST() is just looking in the CFG and if there is missing any of the available option it uses a default one, i left it as 1 value to enable it in that case, i don't know if the unrealircd team agree with that or not.

Reply:
Yeah, these set the defaults. I usually put them in a config_defaults() function to make it more clear. In this case it has a clear comment above it though: ```/* Defaults */```. Perhaps you missed it because of the short context in the diff from GitHub.